### PR TITLE
added a new step to navigate to odim dir before saving proxy config file in ACI plugin deployment procedure

### PR DESCRIPTION
**Pull request by sunithasomanna**
added a new step to navigate to odim dir before saving proxy config file in ACI plugin deployment procedure in ACI readme

### CODE REVIEW

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -171,27 +171,33 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+8. Navigate to the `ODIM` directory.
+
+   ```
+   cd ODIM
+   ```
+
+9. Save the Cisco ACI plugin Docker image on the deployment node at `~/plugins/aciplugin`.
 
    ```
    docker save aciplugin:1.0 -o ~/plugins/aciplugin/aciplugin.tar
```
Comment (sukhwinder20) [inline README.md:183]: Change aciplugin:1.0 to aciplugin:2.0

Comment (sukhwinder20) [inline README.md]: Change aciplugin:1.0 to aciplugin:2.0

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
Please address the comment

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: done

**Comment by sunithasomanna** [comment]
> Please address the comment

done

**Review by sukhwinder20 — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**